csrc/scheduler/matmul.cpp

```diff
@@ -701,6 +701,12 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
   cc->axis(4)->parallelize(ParallelType::TIDz);
   cc->axis(5)->parallelize(ParallelType::TIDy);
 
+  scheduler_utils::parallelizeAllLike(
+      cc,
+      -1,
+      {acr, bcr, ab, bb, a, b},
```

Comment:
Were these tensors not parallelized before?

Reply:
Yes, correct, but I don't think this is a problem, because their parallel type should be promoted during lowering.

Reply:
So this is not related to vectorization? Just to make it more explicit?

Reply:
Correct